### DISCUSSION TITLE
Ship API returning error

### DISCUSSION THREAD

**Discussion by tgreenhaw** [Sellbrite/Sellbrite-API#38] [Q&A] [answered]
We are suddenly getting errors on most orders when we mark them as shipped. On order errors we get "SB Order Seq 13859; Exception: The remote server returned an error: (422) Unprocessable Content."
We have examined what we are sending and don't see any differences between orders that work (e.g. sb order sequence 13905) and orders that do not (e.g. sb order sequence 13899).
Can you help asap to let us know why we are getting this error from the Sellbrite API

this is what we are sending to the api: --does not work
{ "shipment": {"carrier_name":"UPS","items":[{"quantity":"1","sku":"141580"}],"sb_order_seq":"13899","shipping_method":"Standard","tracking_number":"1Z6A307F0300874942","warehouse_uuid":"dd79951d-1a7e-421e-86ac-11b6425d6c86"} }
--works
{ "shipment": {"carrier_name":"UPS","items":[{"quantity":"1","sku":"238341"}],"sb_order_seq":"13858","shipping_method":"Standard","tracking_number":"1Z6430870301178126","warehouse_uuid":"28a6ff38-bc96-4d37-a9cb-81be8df4864a"} }

**Comment by rrincon-godaddy** [comment]
Do we have the merchant ID of these examples you mentioned? It'll be helpful to find the reason of the issue

**Comment by rrincon-godaddy** [comment]
And do you know what endpoint is returning the 422 error?

**Comment by tgreenhaw** [comment]
the endpoint is https://api.sellbrite.com/v1/

**Comment by tgreenhaw** [comment]
I'm not sure what you mean by merchant ID.

Our account token is 1ba163ae-e517-4092-b878-571580ec1c51 and Company Name
is Dry 24 Water Restoration LLC

We are experiencing the issue for both Amazon and Walmart so I assume you
don't mean their merchant IDs

**Comment by rrincon-godaddy** [comment]
Hi, I did some tests but I couldn't find an answer for the bug yet. Do you know if these examples you added created a shipment and, if created, do you know the ID of shipments?

**Comment by tgreenhaw** [comment]
One of them did and one of them did not as mentioned in the previous message. I don;t know what you mean by ID of shipments. Here are 33 more orders that will not accept the API request:
14156
14169
14166
14165
14164
14161
14158
14155
14150
14148
14146
14145
14143
14142
14141
14140
14138
14139
14135
14133
14128
14126
14131
14122
14123
14116
14115
14113
14112
14105
14104
14101
14102

**Comment by rengler-godaddy** [comment]
Hey tgreenhaw, 

We believe this issue was related to an update we made to your platform that we have since reverted. We rolled back the change on 4/30 around 12:00 PM PDT. Can you let us know if you’re still seeing an issue?

**Comment by tgreenhaw** [comment, marked as answer]
Yes, everything seems fine now.On May 2, 2024, at 12:53 PM, Jordan ***@***.***> wrote:﻿
Hey @tgreenhaw,
We believe this issue was related to an update we made to your platform that we have since reverted. We rolled back the change on 4/30 around 12:00 PM PDT. Can you let us know if you’re still seeing an issue?

**Reply by rengler-godaddy** [reply]
Sounds good, if you see any further issues let us know!
